packages/tailwind-config/src/tailwind.config.ts

```diff
@@ -14,31 +14,36 @@ const makePrimaryColor: any =
     return 'hsl(' + h + s + l + (opacityValue ? ` / ${opacityValue}` : '') + ')';
   };
 
-/**
- * We explicitly do not use `import { createRequire } from 'node:module'` and
- * `const require = createRequire(import.meta.url)` because it works even without it.
- * E.g. storybook complains about cannot found `module` package
- */
-const componentsPackageJson = require.resolve('@theguild/components/package.json', {
-  /**
-   * Paths to resolve module location from CWD. Without specifying, it picks incorrect
-   * `@theguild/components`, also must be relative
-   */
-  paths: [process.cwd()],
-});
+function getComponentsPatterns() {
+  try {
+    /**
+     * We explicitly do not use `import { createRequire } from 'node:module'` and
+     * `const require = createRequire(import.meta.url)` because it works even without it.
+     * E.g. storybook complains about cannot found `module` package
+     */
+    const componentsPackageJson = require.resolve('@theguild/components/package.json', {
+      /**
+       * Paths to resolve module location from CWD. Without specifying, it picks incorrect
+       * `@theguild/components`, also must be relative
+       */
+      paths: [process.cwd()],
+    });
 
-const componentsPattern = path.relative(
-  process.cwd(),
-  path.posix.join(componentsPackageJson, '..', 'dist/**/*.js'),
-);
+    return [
+      path.relative(process.cwd(), path.posix.join(componentsPackageJson, '..', 'dist/**/*.js')),
+    ];
+  } catch {
+    return [];
```

Comment:
```suggestion
    console.warn("Can't find `@theguild/components` package.")
    return [];
```
let's add `console.warn` too